themes/classic/templates/catalog/listing/product-list.tpl

```diff
@@ -36,6 +36,8 @@
         {include file='catalog/_partials/subcategories.tpl' subcategories=$subcategories}
       {/if}
     {/block}
+    
+    {hook h="displayHeaderCategory"}
 
     <section id="products">
```

Comment:
```suggestion
    
    <section id="products">

        {hook h="displayHeaderCategory"}
```

Reply:
If it's a headerCategory, it must be in the section like the footer. Not outside.

Reply:
@Progi1984 It's correct, look at it again
https://user-images.githubusercontent.com/6097524/102469467-13961c00-4053-11eb-907e-5a1681f6c8c3.JPG

Reply:
Sorry. I mistaked of section in my code reading. Ok for me.